src/Raven.Server/Commercial/SetupManager.cs

```diff
@@ -745,6 +784,24 @@ private static string GetServerUrlFromCertificate(X509Certificate2 cert, SetupIn
             }
         }
 
+        public static void RegisterClientCertInOs(Action<IOperationProgress> onProgress, SetupProgressAndResult progress, X509Certificate2 clientCert)
```

Comment:
Tried this on windows + chrome only.

Reply:
This should be an option in the UI, defaults to yes (RTM)